src/Microsoft.ML.Data/Evaluators/Metrics/RankingMetrics.cs

```diff
@@ -15,7 +15,7 @@ public sealed class RankingMetrics
     {
         /// <summary>
         /// Array of normalized discounted cumulative gains where i-th element represent NDCG@i.
-        /// <image src="https://github.com/dotnet/machinelearning/tree/master/docs/images/NDCG.png"></image>
+        /// <image src="https://github.com/dotnet/machinelearning/blob/3cdd3c8b32705e91dcf46c429ee34196163af6da/docs/images/NDCG.png?raw=true"></image>
```

Comment:
Is it ideal to point to a fixed commit? What if we change the figure but forget to update this link?

Reply:
I think we want to point to a fixed commit. If we point to /main/, our old docs will break if we move the image's location within the repo. 

We keep old versions of the docs for each release:
* https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.rankingmetrics?view=ml-dotnet-1.1.0
* https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.rankingmetrics?view=ml-dotnet
* https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.rankingmetrics?view=ml-dotnet-preview

The repo is expected to evolve over time. The better solution is having the docs keep track of its own images (which is the 1st item under 'ssues NOT fixed by this PR').

Hopefully the person modifying the images first looks in the code for where they are used. They should then see that they have to update the pinned commit.